packages/read/src/file-system/file-system.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from 'vitest';
+import { FileSystem } from './file-system';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { formatGraphForTestSnapshot } from '../common';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+describe('FileSystem provider', () => {
+  it('should produce the expected graph', async () => {
+    const graph = await FileSystem.read({
+      path: path.resolve(__dirname, '../../../', 'draw', 'example', 'notes'),
+    });
+
+    expect(formatGraphForTestSnapshot(graph)).toMatchFileSnapshot(
```

Comment:
No suggestions, just thought the testing approach of building a graph from the filesystem, and using the same graph against the Github API was pretty clever! Good work!